packages/devel/glib/package.mk

```diff
@@ -18,45 +18,46 @@
 ################################################################################
 
 PKG_NAME="glib"
-PKG_VERSION="2.53.5"
-PKG_SHA256="991421f41a4ed4cc1637e5f9db0d03fd236d2cbd19f3c5b097a343bec5126602"
+PKG_VERSION="2.54.1"
+PKG_SHA256="7413aedbfd3108aceea16e8b1b487d557066be3f8ce34dd3be308128c5caf5b7"
 PKG_ARCH="any"
 PKG_LICENSE="LGPL"
 PKG_SITE="http://www.gtk.org/"
-PKG_URL="http://ftp.gnome.org/pub/gnome/sources/glib/${PKG_VERSION%.*}/$PKG_NAME-$PKG_VERSION.tar.xz"
-PKG_DEPENDS_TARGET="toolchain zlib libffi Python2:host util-linux"
-PKG_DEPENDS_HOST="libffi:host pcre:host"
+PKG_URL="https://github.com/GNOME/glib/archive/$PKG_VERSION.tar.gz"
+PKG_DEPENDS_TARGET="toolchain zlib libffi Python3:host util-linux"
+PKG_DEPENDS_HOST="Python3:host libffi:host pcre:host"
 PKG_SECTION="devel"
 PKG_SHORTDESC="glib: C support library"
 PKG_LONGDESC="GLib is a library which includes support routines for C such as lists, trees, hashes, memory allocation, and many other things."
 
 PKG_IS_ADDON="no"
```

Comment:
`PKG_IS_ADDON="no"` could be dropped
at line 4 you have `Copyright (C) 2016-     Team LibreELEC`  should be `2016-present`
all other new packages have still `Copyright (C) 2016 Team LibreELEC` too